packs/smart_items/assets/sign_neon_arrow/composite.json

```diff
@@ -72,6 +74,7 @@
       "data": {
         "513": {
           "json": {
+            "basicViewId": "text1",
```

Comment:
the `basicViewId` is used by the `Inspector::Config` component to link with the Actions or Trigger, not required for this case

Reply:
I just removed this on all items